app/screens/accept-t-and-c/accept-t-and-c.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from "react"
+import { View } from "react-native"
+
+import { GaloyPrimaryButton } from "@app/components/atomic/galoy-primary-button"
+import { useI18nContext } from "@app/i18n/i18n-react"
+import { RootStackParamList } from "@app/navigation/stack-param-lists"
+import { RouteProp, useNavigation, useRoute } from "@react-navigation/native"
+import { StackNavigationProp } from "@react-navigation/stack"
+import { Text, makeStyles } from "@rneui/themed"
+
+import { GaloySecondaryButton } from "@app/components/atomic/galoy-secondary-button"
+import InAppBrowser from "react-native-inappbrowser-reborn"
+import { Screen } from "../../components/screen"
+import { PhoneLoginInitiateType } from "../phone-auth-screen"
+import { DeviceAccountModal } from "../get-started-screen/device-account-modal"
+
+export const AcceptTermsAndConditionsScreen: React.FC = () => {
+  const styles = useStyles()
+
+  const navigation =
+    useNavigation<StackNavigationProp<RootStackParamList, "acceptTermsAndConditions">>()
+
+  const { LL } = useI18nContext()
+
+  const route = useRoute<RouteProp<RootStackParamList, "acceptTermsAndConditions">>()
+  const { flow } = route.params || { flow: "phone" }
+
+  const [confirmationModalVisible, setConfirmationModalVisible] = React.useState(false)
+  const openConfirmationModal = () => setConfirmationModalVisible(true)
+  const closeConfirmationModal = () => {
+    setConfirmationModalVisible(false)
+  }
+
+  const action = () => {
+    if (flow === "phone") {
+      navigation.navigate("phoneFlow", {
+        screen: "phoneLoginInitiate",
+        params: {
+          type: PhoneLoginInitiateType.CreateAccount,
+        },
+      })
+    } else {
```

Comment:
What about the trial flow? Also how do you leave the screen if the action just opens the modal?

Reply:
does this address #1 ? https://github.com/GaloyMoney/galoy-mobile/pull/3218/commits/2fd8b3a08c0aedcf6a8f1458613b7d09fbc8e3a4

on the second point, there is not much logic changes, I just moved the logic from getStartedScreen to here, because the logic is within a modal. but the action to move to other screen were already in the modal.

Reply:
Nevermind, I was confused by the naming. I thought the confirmation modal was to confirm the T and C not confirm device account. I would probably have the fallback be navigate to the home screen instead of alerting.

Reply:
Maybe out of scope for this PR but we probably want some audit trail that terms and conditions were accepted.